e2e/scripts/pageHelpers.js

```diff
@@ -229,10 +263,64 @@ export const registerShopper = async ({page, userCredentials, isMobile = false})
     await tokenResponsePromise
     expect((await tokenResponsePromise).status()).toBe(200)
 
-    await expect(page.getByRole('heading', {name: /Account Details/i})).toBeVisible()
+    await page.waitForLoadState('networkidle', { timeout: 10000 })
```

Comment:
this option is discouraged. Can we find the alternative?
![image](https://github.com/user-attachments/assets/b06d59d2-9ab5-4d60-849e-d0244ac2eef0)

Reply:
wait for request/response is a better option: https://github.com/SalesforceCommerceCloud/storefrontdata